unison-src/transcripts/fix942.output.md

```diff
@@ -94,7 +94,7 @@ test> t1 = if z == 3 then [Fail "nooo!!!"] else [Ok "great"]
   
     ⍟ These new definitions are ok to `add`:
     
-      t1 : [builtin.Test.Result]
+      t1 : [Result]
```

Comment:
I have to keep reminding myself that this shortening is only happening because there's only one `Result` type in the namespace. (Right?)

Reply:
That's right.